gammapy/cube/models.py

```diff
@@ -437,3 +438,49 @@ def compute_npred(self):
         if self.background:
             npred.data += self.background.data
         return npred.data
+
+
+class SkyMap3d(object):
+    """Cube sky map template model.
+
+    Parameters
+    ----------
+    map : `~gammapy.map.Map`
+        Map template
+    norm : `~astropy.units.Quantity`
+        Norm parameter (multiplied with map values)
+    meta : dict, optional
+        Meta information, meta['filename'] will be used for serialization
+    """
+
+    def __init__(self, map, norm=1, meta=None):
+        self._map = map
```

Comment:
You might add a test to check that the Map has a single axis of type energy

Reply:
Yes, you could put something like this as input map validation:
```
if len(map.geom.axes) != 0:
    raise ValueError('Need a map with an energy axis')

axis = map.geom.axes[0]
if axis.name != 'energy':
    raise ValueError('Need a map with axis of name "energy"')

if axis.node_type != 'center':
    raise ValueError('Need a map with energy axis node_type="center"')
```
Usually we don't do so much input validation, but given that likely people will try to pass 2D maps or 3D maps that aren't appropriate here, I agree it's a good addition to check some things.